### PR TITLE
Change the auto play of the video's

### PR DESCRIPTION
**Pull request by shavit2310**
update the autoplay

### CODE REVIEW

File: kikar_hamedina/core/templates/core/facebook_status.html [hunk 1]
```diff
@@ -88,14 +88,14 @@
...
-                                            <embed>
+                                            <embed autoplay="false">
                                                 <source src="{{ status.attachment.source }}" type="video/mp4">
                                                 <source src="{{ status.attachment.source }}" type="video/ogg">
                                             {% trans 'Apologies, your broswer does not support displaying videos.' %}
                                             </embed>
                                         </object>
                                     {% else %}
-                                     <video width="auto" height="auto" autoplay controls>
+                                     <video width="auto" height="auto">
```
Comment (yotammanor) [inline kikar_hamedina/core/templates/core/facebook_status.html]: Are you sure we don't need the "controls" attribute?

File: kikar_hamedina/core/urls.py [hunk 2]
```diff
@@ -17,6 +17,7 @@
 from core.models import MEMBER_MODEL, PARTY_MODEL
 from core.rss_feeds import LatestStatusesRSSFeed, PartyRSSFeed, MemberRSSFeed, KeywordsByUserRSSFeed, CustomQueryRSSFeed
 from django.contrib.sitemaps.views import sitemap
+#from . import video_type
```
Comment (yotammanor) [inline kikar_hamedina/core/urls.py]: Don't leave around commented out unused imports, just delete them.

File: kikar_hamedina/core/urls.py [hunk 3]
```diff
@@ -118,4 +119,4 @@
                            name='django.contrib.sitemaps.views.sitemap'),
                        ) + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
 
-urlpatterns = format_suffix_patterns(urlpatterns)
+urlpatterns = format_suffix_patterns(urlpatterns)
```
Comment (yotammanor) [inline kikar_hamedina/core/urls.py]: re-add the line end, it is pep8 to leave it.

File: kikar_hamedina/facebook_feeds/models.py [hunk 4]
```diff
@@ -441,6 +441,17 @@ def is_youtube_video(self):
         return False
 
     @property
+    def video_source_type(self):
```
Comment (yotammanor) [inline kikar_hamedina/facebook_feeds/models.py]: Are we using this? Is this here on purpose? if so, how does it relate to the pull request?

File: kikar_hamedina/core/templates/core/facebook_status.html (hunk 1 above)
Comment (shavit2310) [inline kikar_hamedina/core/templates/core/facebook_status.html]: This is a mistake. it is not relevant. pls do not update the main repository with this file.

File: kikar_hamedina/core/urls.py (hunk 2 above)
Comment (shavit2310) [inline kikar_hamedina/core/urls.py]: This is a mistake. it is not relevant. pls do not update the main repository with this file.

File: kikar_hamedina/core/urls.py (hunk 3 above)
Comment (shavit2310) [inline kikar_hamedina/core/urls.py]: This is a mistake. it is not relevant. pls do not update the main repository with this file.

File: kikar_hamedina/facebook_feeds/models.py (hunk 4 above)
Comment (shavit2310) [inline kikar_hamedina/facebook_feeds/models.py]: This is a mistake. it is not relevant. pls do not update the main repository with this file.

File: kikar_hamedina/core/templates/core/facebook_status.html (hunk 1 above)
Comment (shavit2310) [inline kikar_hamedina/core/templates/core/facebook_status.html]: only this file need to be updated:"kikar_hamedina/core/templates/core/facebook_status.html", ar are mistake.

**Comment by shavit2310** [comment]
Only the file: kikar_hamedina/core/templates/core/facebook_status.html should access to this repository and merge pull requests.
the rest 3 files- should'nt.

**Comment by yotammanor** [comment]
fixed in #264 .

**Pull request closed without merging**